### DISCUSSION TITLE
How long did it take to construct The Empire State Building?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#376] [Galaxy Trivia] [answered]
**Hint:** use days for precision

**Comment by thuongtruong1009** [comment]
The Empire State Building was constructed relatively quickly, taking only one year and 45 days to complete. Construction began on March 17, 1930, and the building officially opened on May 1, 1931. At the time, it was the tallest building in the world, standing at 1,454 feet (443.2 meters) tall.

**Comment by peterbaccaro** [comment]
1 year and 45 days.

**Comment by Roberto-Larivera** [comment, marked as answer]
L'Empire State Building è stato costruito in poco più di un anno e mezzo. I lavori iniziarono il 17 marzo 1930 e furono completati il 1º maggio 1931, per un totale di circa 13 mesi di costruzione effettiva. In totale, sono stati impiegati circa 3.400 lavoratori per la costruzione dell'edificio, che conta 102 piani e un'altezza di 443,2 metri (1.454 piedi) dalla base all'antenna sulla cima. All'epoca della sua costruzione, l'Empire State Building era l'edificio più alto del mondo.

Giorni in totale: 410

**Reply by ParanoidUser** [reply]
Bravo, in 410 days! 👏👏
